filament/include/filament/RenderableManager.h

```diff
@@ -340,6 +342,74 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
+         *
+         * @param indicesAndWeightsArray pairs of bone index and bone weight for all vertices and primitives sequentially
+         * @param count number of all pairs, must be a multiple of vertexCount by primitiveCount
+         * @param offset specifies where in the indicesAndWeights buffer to start reading (expressed as a number of pairs)
+         *
+         * @return Builder reference for chaining calls.
+         */
+        Builder& boneIndicesAndWeights(math::float2 const* indicesAndWeightsArray, size_t count, size_t offset = 0);
+
+        /**
+         * Define bone indices and weights for vertex skinning. The first pair value defines index of the bone and the
+         * second value is the bone weight. The pairs substitute \c BONE_INDICES and the \c BONE_WEIGHTS
+         * defined in the VertexBuffer. Both ways of indices and weights definition must not be combined in one primitive.
+         * Number of pairs is not limited to 4 bones per vertex.
+         * All bone weights of one vertex should sum to one. Otherwise they will be normalized.
+         * Data doesn't have to be rectangular and number of pairs per vertices of primitive can be variable.
+         * The vector of the vertices contains the vectors of the pairs
+         *
+         * @param primitiveIndex zero-based index of the primitive, must be less than the primitive count passed to Builder constructor
+         * @param indicesAndWeightsVectors pairs of bone index and bone weight for all vertices of the primitive sequentially
+         *
+         * @return Builder reference for chaining calls.
+         */
+        Builder& boneIndicesAndWeights(size_t primitiveIndex,
+               utils::FixedCapacityVector<utils::FixedCapacityVector<filament::math::float2>> const &indicesAndWeightsVector);
+        /**
+         * Define bone indices and weights for vertex skinning. The first pair value defines index of the bone and the
+         * second value is the bone weight. The pairs substitute \c BONE_INDICES and the \c BONE_WEIGHTS
+         * defined in the VertexBuffer. Both ways of indices and weights definition must not be combined in one primitive.
+         * Number of pairs is not limited to 4 bones per vertex.
+         * All bone weights of one vertex should sum to one. Otherwise they will be normalized.
+         * Data doesn't have to be rectangular and number of pairs per vertices and primitives can be variable.
+         * The vector of the primitives contains the vectors of the vertices with the vectors of the pairs
+         *
+         * @param indicesAndWeightsVectors pairs of bone index and bone weight for all vertices and primitives sequentially
+         *
+         * @return Builder reference for chaining calls.
+         */
+        Builder& boneIndicesAndWeights(
+                utils::FixedCapacityVector< utils::FixedCapacityVector< utils::FixedCapacityVector<filament::math::float2>>> const &indicesAndWeightsVectors) noexcept;
```

Comment:
Same as above, is this really needed? Caller could call in a loop?

Reply:
Ok, I'll remove it for simplicity.